README.md

```diff
@@ -259,9 +259,170 @@ Otherwise the client will return an error `err`:
 </table>
 </details>
 
+### Letter
+
+```javascript
+notifyClient
+	.sendLetter(templateId, personalisation, reference)
+    .then(response => console.log(response))
+    .catch(err => console.error(err))
+;
+```
+
+where `personalisation` is
+
+```javascript
+personalisation={
+    'address_line_1': 'The Occupier',  # required
+    'address_line_2': '123 High Street', # required
+    'address_line_3': 'London',
+    'postcode': 'SW14 6BH',  # required
+
+    ... # any other optional address lines, or personalisation fields found in your template
+},
+```
+
+<details>
+<summary>
+Response
+</summary>
+
+If the request is successful, `response` will be a `Dictionary`:
```

Comment:
Javascript doesn’t have dictionaries. I think `object` is the right term.

Reply:
ok